### PR TITLE
LightningStruck Event

### PR DESCRIPTION
**Pull request by gegy1000**

### CODE REVIEW

File: patches/net/minecraft/world/biome/Biome.java.patch [hunk 1]
```diff
@@ -1,29 +1,29 @@
---- /net/minecraft/world/biome
```
Comment (darkhax) [inline patches/net/minecraft/world/biome/Biome.java.patch]: Why are you patching biomes for this?

Comment (iLexiconn) [inline patches/net/minecraft/world/biome/Biome.java.patch]: He's not, just a line ending error

File: src/main/java/xyz/openmodloader/event/impl/BiomeEvent.java [hunk 2]
```diff
@@ -1,158 +1,158 @@
-package xyz.openmodloader.event.impl;
```
Comment (darkhax) [inline src/main/java/xyz/openmodloader/event/impl/BiomeEvent.java]: Same here lol

File: src/main/java/xyz/openmodloader/event/impl/EntityEvent.java [hunk 3]
```diff
@@ -213,4 +214,36 @@ public Unmount(Entity entity, Entity riding) {
...
+
+    /**
+     * This event is fired when an entity is struck by lightning.
+     */
+    public static class LightningStruck extends EntityEvent {
+
+        /**
+         * The lightning bolt that struck the entity
+         */
+        protected EntityLightningBolt lightningBolt;
```
Comment (iLexiconn) [inline src/main/java/xyz/openmodloader/event/impl/EntityEvent.java]: make final

File: src/test/java/xyz/openmodloader/test/OMLTestMod.java [hunk 4]
```diff
@@ -1,51 +1,40 @@
...
 import net.minecraft.util.text.TextComponentString;
 import net.minecraft.world.WorldServer;
+import org.lwjgl.input.Keyboard;
 import xyz.openmodloader.OpenModLoader;
 import xyz.openmodloader.client.gui.GuiModInfo;
 import xyz.openmodloader.client.gui.GuiModList;
 import xyz.openmodloader.config.Config;
-import xyz.openmodloader.event.impl.ArmorEvent;
+import xyz.openmodloader.event.impl.*;
 import xyz.openmodloader.event.impl.BiomeEvent.BiomeColor;
```
Comment (darkhax) [inline src/test/java/xyz/openmodloader/test/OMLTestMod.java]: If you are staring them, no need to import the biome event.

**Pull request closed without merging**